### ISSUE TITLE
kau-logo-tryck graphic file(s) missing

### ISSUE THREAD

**Issue by Sharparam** [ZjMNZHgG5jMXw/kauthesis#3]
The `kautitle` package cannot find the graphic file(s) for `kau-logo-tryck` since they are not included anywhere.

Adding them manually from another source (KAU has a lab report template that includes the file) makes it work. I placed a PDF and EPS version in `$TEXMFHOME/tex/generic/images/`.

The title page also has very oddly chosen colours that are incredibly difficult to read.

MWE for title page:

```latex
\documentclass{kaucollection}
\usepackage{kautitle}

\title{Test title}
\author{Adam Hellberg}
\subject{Test subject \#01}
\institute[TGDDI]{Department of Computer Science, Karlstad University}
\place{Karlstad}
\date{December 2016}

\begin{document}
\maketitle
\frontmatter
\begin{abstract}
This is an abstract.
\end{abstract}
\tableofcontents
\mainmatter
\section{First section}
\subsection{First subsection}
\end{document}
```

[This PDF file](https://github.com/ZjMNZHgG5jMXw/kauthesis/files/629207/test.pdf) generated from the above MWE displays the issue.

**Comment by ZjMNZHgG5jMXw** [comment]
Hi,

The logo cannot be published here on Github.com unless the copyright applying to it is sorted out. I was not eager to do it, but I made sure in my labs that I supplied the logo within KAU where necessary. Good that you found the correct location in the TEXMF tree to add the logo in your possession.

As to the odd colours, the title page generated by TeX does not serve any better purpose than satisfying supervisors who wouldn't understand that the title page is always created by the publisher. The page is definitely not meant to be published.

That's by the way also the secret behind the fact that the TeX-rendered documents have to start with page no. iii --- the publisher will add, besides the outer title page, two pages with the "half title" and meta information about your work. Thus, your content starts on page 3.

Hope, that clarifies it.

Cheers,
Stefan.
